### PR TITLE
Gantt: add sorting section

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: concepts/05 UI Components/Gantt/20 Sort Data/20 Sort in Code.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
+Set a column's [sortOrder](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortOrder) property to sort data in the required order. You can also specify the [sortIndex](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortIndex) property for this column to sort data by multiple columns. If the [sortIndex](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortIndex) property is not specifies, each sorted column gets a sort index according to its position in the [columns](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/) array. To show column sort indexes in the column headers, set the [showSortIndexes](/api-reference/_hidden/dxGanttSorting/showSortIndexes.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#showSortIndexes') property to **true**.
```
Comment (LenaDv) [inline concepts/05 UI Components/Gantt/20 Sort Data/20 Sort in Code.md]: Set a column's sortOrder property... - to which value?

File: concepts/05 UI Components/Gantt/20 Sort Data/Sort Data.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
+The Gantt component allows users to sort data. 
+
+Use the following properties to specify whether users can sort column data:
+
+- [mode](/api-reference/_hidden/dxGanttSorting/mode.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode') - For all columns.
+- **column.**[mode](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode) - For an individual column. Note that this property is in effect if the [mode](/api-reference/_hidden/dxGanttSorting/mode.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode') property is not set to 'none'.
+
+Use the **sorting**.[mode](/api-reference/_hidden/dxGanttSorting/mode.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode') property to specify the sorting mode.
```
Comment (LenaDv) [inline concepts/05 UI Components/Gantt/20 Sort Data/Sort Data.md]: "...to specify the sorting mode." - Does the sorting.mode property specify whether the Gantt component allows sorting data by multiple columns or by a single column only?

File: concepts/05 UI Components/Gantt/20 Sort Data/20 Sort in Code.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
+Use a column's [sortOrder](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortOrder) property to specify sort order of the column values. You can also specify the [sortIndex](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortIndex) property for this column to sort data by multiple columns. If the [sortIndex](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortIndex) property is not specifies, each sorted column gets a sort index according to its position in the [columns](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/) array. To show column sort indexes in the column headers, set the [showSortIndexes](/api-reference/_hidden/dxGanttSorting/showSortIndexes.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#showSortIndexes') property to **true**.
```
Comment (arminal) [inline concepts/05 UI Components/Gantt/20 Sort Data/20 Sort in Code.md]:
```suggestion
Use a column's [sortOrder](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortOrder) property to specify the sort order for column values. You can also specify the [sortIndex](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortIndex) property for this column to sort data by multiple columns. If the [sortIndex](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/#sortIndex) property is not specified, each sorted column gets a sort index according to its position in the [columns](/Documentation/ApiReference/UI_Components/dxGantt/Configuration/columns/) array. To show column sort indexes in the column headers, set the [showSortIndexes](/api-reference/_hidden/dxGanttSorting/showSortIndexes.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#showSortIndexes') property to **true**.
```

File: concepts/05 UI Components/Gantt/20 Sort Data/20 Sort in Code.md [hunk 4]
```diff
@@ -0,0 +1,311 @@
...
+
+The following code sorts data by the "Start" and then by the "End" column:
```
Comment (arminal) [inline concepts/05 UI Components/Gantt/20 Sort Data/20 Sort in Code.md]:
```suggestion
The following code sorts data by the "Start" and "End" columns:
```

File: concepts/05 UI Components/Gantt/20 Sort Data/20 Sort in Code.md [hunk 5]
```diff
@@ -0,0 +1,311 @@
...
+        })
+        .Sorting(s => {
+            s.Mode(GridSortingMode.Multiple)
+        })
+        // ...
+    )
+
+---
+
+Use the [ascendingText](/api-reference/_hidden/dxGanttSorting/ascendingText.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#ascendingText'), [descendingText](/api-reference/_hidden/dxGanttSorting/descendingText.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#descendingText'), and the [clearText](/api-reference/_hidden/dxGanttSorting/clearText.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#clearText') options to specify text for the corresponding context menu items.
```
Comment (arminal) [inline concepts/05 UI Components/Gantt/20 Sort Data/20 Sort in Code.md]:
```suggestion
Use the [ascendingText](/api-reference/_hidden/dxGanttSorting/ascendingText.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#ascendingText'), [descendingText](/api-reference/_hidden/dxGanttSorting/descendingText.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#descendingText'), and the [clearText](/api-reference/_hidden/dxGanttSorting/clearText.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#clearText') options to specify text for corresponding context menu items.
```

File: concepts/05 UI Components/Gantt/20 Sort Data/Sort Data.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
+The Gantt component allows users to sort data. 
+
+Use [mode](/api-reference/_hidden/dxGanttSorting/mode.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode') property to specify whether users can sort data by a single or multiple columns.
```
Comment (arminal) [inline concepts/05 UI Components/Gantt/20 Sort Data/Sort Data.md]:
```suggestion
Use the [mode](/api-reference/_hidden/dxGanttSorting/mode.md '/Documentation/ApiReference/UI_Components/dxGantt/Configuration/sorting/#mode') property to specify whether users can sort data by single or multiple columns. 
```

**Comment by arminal** [comment]
#GrammarReview

Anna, you must, must, must proofread your docs before sending them to an editor. This was a pretty short doc, but it had too many errors that you should have caught yourself.

**Review by LanaDX — APPROVED** [review]

**Pull request merged**